src/pages/Landing/components/EventsCalendarView.tsx

```diff
@@ -30,15 +29,42 @@ type AppointmentProps = {
 
 const Appointment = ({ children, data }: AppointmentProps) => {
   const theme = useTheme();
+  const [anchorEl, setAnchorEl] = useState<null | HTMLElement>(null);
+  const open = Boolean(anchorEl);
+
+  const handleClick = (event: React.MouseEvent<HTMLElement>) => {
+    setAnchorEl(anchorEl ? null : event.currentTarget);
+  };
+
+  const handleClickAway = () => {
+    setAnchorEl(null);
+  };
 
   const getColor = (event: EventCompact) =>
     theme.palette.colors[event.organizer?.slug.toLowerCase() === Groups.NOK.toLowerCase() ? 'nok_event' : 'other_event'];
   return (
-    <Link to={`${URLS.events}${data.id}/${urlEncode(data.title)}/`}>
-      <Appointments.Appointment data={data} draggable={false} resources={[]} style={{ backgroundColor: getColor(data as unknown as EventCompact) }}>
-        {children}
-      </Appointments.Appointment>
-    </Link>
+    <div>
```

Comment:
Må det være en div rundt eller kan du bruke <>?
```suggestion
    <>
```